src/linear_algebra/matrix/determinant.lean

```diff
@@ -104,11 +104,21 @@ lemma det_unique {n : Type*} [unique n] [decidable_eq n] [fintype n] (A : matrix
   det A = A (default n) (default n) :=
 by simp [det_apply, univ_unique]
 
+lemma det_eq_elem_of_subsingleton [subsingleton n] (A : matrix n n R) (k : n) :
+  det A = A k k :=
+begin
```

Comment:
I would have thought you can prove this much faster with `det_unique` and `unique.mk'`